tools/Makefile-tools.am

```diff
@@ -22,8 +22,10 @@ coverage:
 	@echo "file://$(abs_top_builddir)/tools/coverage/index.html"
 endif
 
-TESTS += \
-	tools/test-static-code \
-	tools/test-bots
+TESTS += tools/test-static-code
+
+if WITH_PYTHON3
+TESTS += tools/test-bots
```

Comment:
this is a bit of a shame.  do you think it might be worth it to try harder to keep this working on py2 systems, or are we basically okay with putting the non-py3 world on life support?

two options:

1. .py.in some stuff
2. create a symlink in-tree and use it as the interpreter in our bilingual scripts

we could also just give up on py2.  i'm rather OK with that, if you don't think it's a big loss.

Reply:
There is little point in keeping the bots code and tests for py2. bots/ is a rather independent thing (it should even be split out into a separate project at some point), and is not being shipped in releases at all, i. e. this is totally irrelevant for downstream testing. Thus I think any kind of .py.in, symlinks etc. would be overkill, and just make it harder to run stuff from a clean tree.

Conceptually, test-bots should even move to bots/, it has no dependencies on test/ and all the actual tests are already in bots/tasks/. I'll look into moving it there.

Reply:
Done in another separate commit.